internal/bminventory/inventory.go

```diff
@@ -2041,7 +2041,12 @@ func (b *bareMetalInventory) setDefaultUsage(cluster *models.Cluster) {
 	b.setUsage(swag.StringValue(cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone,
 		usage.HighAvailabilityModeUsage, nil, usages)
 	b.setProxyUsage(&cluster.HTTPProxy, &cluster.HTTPProxy, &cluster.NoProxy, usages)
-	b.setOperatorsUsage(cluster.MonitoredOperators, []*models.MonitoredOperator{}, usages)
+	olmOperators := funk.Filter(cluster.MonitoredOperators, func(op *models.MonitoredOperator) bool {
```

Comment:
Why are you filtering only the OLM operator?

Reply:
Because we are not interested in the console. It comes by default